test/continuous/test_td3.py

```diff
@@ -120,4 +120,6 @@ def stop_fn(x):
 
 
 if __name__ == '__main__':
+    from tianshou.utils import pre_compile
+    pre_compile()  # exclude compilation time to get the correct train_speed
```

Comment:
That's a bit strange for newcomers. Maybe ``pre_compile `` can be called during tianshou.__init__ .

Reply:
I think so, but this conflicts with @duburcqa https://github.com/thu-ml/tianshou/pull/193#discussion_r480140514

Reply:
> Then I would rather move _compile at the beginning of the brenchmark, rather than here. I thing it makes more sense, because it is really just a benchmarking issue.

There are many benchmarks in tianshou, but some of the benchmarks (in ``test`` folder) are also examples for beginners. Therefore, I think it is not a good solution to add pre-compile for every benchmark. Pre-compile in ``tianshou.__init__.py`` should be enough.

@duburcqa

Reply:
I get your point but it is an anti-pattern. JIT is for jit. If precompile is actually able to precompile all you need, then I would use, it works just the same as jit and it is actually intenses to do what you are looking for.

Reply:
Well AOT is not exactly as portable as JIT in practice because of some limitations. You can put it in init if you want but I really don't like the idea. I would rather keep it as it is right now.

Reply:
But beginners may build up their own code and benchmark and say, "Hey, your provided result is not compatible with my own test", something like #169 and #196.

Reply:
I don't think it is a sad thing. People have never been able to properly benchmark a code, you can see this in all but one in a million problems like this. It is a petty to lower our standard to match people programming knowledge.